doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -289,6 +290,22 @@ Apart from the `status` values defined for subscription in the NGSIv2 specificat
 
 [Top](#top)
 
+## Force Updatation of Entity
+NGSIv2 introduced a new URI param option(forcedUpdate) for updating the entity if the value of the attribute is same as previous value.
```

Comment:
I'd suggest some small modifications. Final text:

As extra URI param option to the ones included in the NGSIv2 specification, Orion implements `forcedUpdate`, than can be used to specify that an update operation have to trigger any matching subscription (and send corresponding notification) no matter if there is an actual attribute update or not. Remember that the default behaviour (i.e. without using the `forcedUpdate` URI param option) is to updated only if attribute is effectively updated.

The following requests can use the `forcedUpdate` URI param option:
...
```

Reply:
Fixed in 6df4217